File: Common/CommonEntities/Auth/UserInfo.cs
using CommonEntities.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static CommonEntities.Enums;

namespace CommonEntities.Auth
{
    public class UserInfo : BaseRequest
    {
        public int Id { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Salt { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public UserInfoStatus UserInfoStatus { get; set; }
        public string MobileNumber { get; set; }
        public int UserSubjectId { get; set; }
    }
}

File: UI/MyInterviewPreparation/ClientApp/src/Entities/UserInfo.ts
import { UserInfoStatus } from "./Enums";
import { BaseRequest } from "./BaseRequest";

export interface UserInfo extends BaseRequest {
    id: number;
    createdDate: string | null;
    updatedDate: string | null;
    email: string;
    password: string;
    salt: string;
    firstName: string;
    lastName: string;
    userInfoStatus: UserInfoStatus;
    mobileNumber: string;
    userSubjectId: number | 0;
}